addon/styles/_prism-overrides.scss

```diff
@@ -32,6 +32,11 @@ code .diff-operator {
   user-select: none;
 }
 
+code .diff-deletion {
+  // don't include removed code in copy-paste
+  user-select: none;
+}
```

Comment:
Actually I’m not sure this does what you want. I think at least in some browsers, this only affects how the selection looks visually, but doesn’t prevent them from being copied if you select the surrounding text like you did in the screenshot. Have you tested it on multiple browsers?

Reply:
I have not, but I just looked it up on caniuse and it seems I need to prefix it. Thanks for bringing this up!